WTWSMS/interface/WtWSMS_decisions_icons.gfx

```diff
@@ -385,4 +393,12 @@ spriteTypes = {
 		name = "GFX_create_priscillian_patriarchy"
 		texturefile = "gfx\\interface\\GFX_create_priscillian_patriarchy.tga"
 	}
+	spriteType = {
+		name = "GFX_peregrinatio"
+		texturefile = "gfx\\interface\\decision_icon_catholic_pilgrimage.dds"
+	}
+	spriteType = {
+		name = "GFX_the_henotikon"
+		texturefile = "gfx\\interface\\decision_icon_write_theory_paper.dds"
+	}
```

Comment:
Good ideas on the icons, although the best would be unique ones. These will serve for now though.